feature-libs/product-configurator/rulebased/styles/_configurator-add-to-cart-button.scss

```diff
@@ -6,15 +6,71 @@
   border-top: solid 1px var(--cx-color-light);
   box-shadow: 0 0 5px var(--cx-color-light);
 
-  .cx-add-to-cart-btn-container,
-  .cx-display-only-btn-container {
-    max-width: 1140px;
-    @include cx-configurator-footer-container();
-    margin-top: 0px;
-
-    button.cx-add-to-cart-btn,
-    button.cx-display-only-btn {
-      @include cx-configurator-footer-container-item();
+  @include forVersion(6) {
+    .cx-add-to-cart-btn-container,
+    .cx-display-only-btn-container {
+      max-width: 1140px;
+      @include cx-configurator-footer-container();
+      margin-top: 0px;
+
+      button.cx-add-to-cart-btn,
+      button.cx-display-only-btn {
+        @include cx-configurator-footer-container-item();
+      }
+    }
+  }
+
+  @include forVersion(6.1) {
+    .cx-add-to-cart-btn-container,
+    .cx-display-only-btn-container {
+      max-width: 1140px;
```

Comment:
no need to include this because it exists in 6.0 already. You only need to include your 6.1 changes in the 6.1 wrapper.

Reply:
Thanks a lot for review!
I have worked in your feedback :)